src/routes/controllers/users.js

```diff
@@ -1,13 +1,15 @@
 import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
 import models from '../../db/models';
```

Comment:
we can import `User` from models on 3 using 
`import { User } from '../../db/models';`